.wuf/generator/templates/Component/Component.spec.tsx

```diff
@@ -3,27 +3,28 @@ import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
 import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
 import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
 import { mount } from 'enzyme';
-import { TPAComponentsWrapper } from '../../test/utils';
 import { {%componentName%}DriverFactory } from './{%ComponentName%}.driver';
-import { {%ComponentName%} } from './';
 import { {%componentName%}TestkitFactory } from '../../testkit';
 import { {%componentName%}TestkitFactory as enzyme{%ComponentName%}TestkitFactory } from '../../testkit/enzyme';
+// import { TPAComponentsProvider, TPAComponentsConfig } from '../TPAComponentsConfig';
+import { {%ComponentName%}, {%ComponentName%}Props } from './';
 
 describe('{%ComponentName%}', () => {
   const createDriver = createUniDriverFactory({%componentName%}DriverFactory);
 
+  const bootstrap = (props: Partial<{%ComponentName%}Props> = {}/*, contextProps: TPAComponentsConfig = {}*/) => {
+    return createDriver(
+      // <TPAComponentsProvider value={contextProps}>
+        <{%ComponentName%} {...props} />
+      // </TPAComponentsProvider>
+    );
+  };
+
   it('should render', async () => {
-    const driver = createDriver(<{%ComponentName%} buttonText="Click Me" />);
+    const driver = bootstrap();
```

Comment:
Why did you change it for that?

Reply:
It's just to give a generic method to be used in all tests.
Instead of copy-pasting `createDriver(<MyComponent />)` in each test.
Especially if it's wrapped with the `TPAComponentsProvider`.

Btw, I see that I forgot to pass props to the `bootstrap` method.
I'll fix that.